hedera-node/src/main/java/com/hedera/services/grpc/controllers/ContractController.java

```diff
@@ -20,18 +20,108 @@
  * ‍
  */
 
-public class ContractController {
+import com.hedera.services.queries.answering.QueryResponseHelper;
+import com.hedera.services.queries.contract.ContractAnswers;
+import com.hedera.services.queries.crypto.CryptoAnswers;
```

Comment:
This import statement is not needed.

Reply:
Ah good catch, I'll remove those in a future PR so you don't need to re-approve 😁  (or if Quan requests changes, will remove the imports when addressing those).